src/Sylius/Bundle/AttributeBundle/Resources/config/services/attribute_types.xml

```diff
@@ -56,5 +56,20 @@
                  form-type="Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\SelectAttributeType"
                  configuration-form-type="Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration\SelectAttributeConfigurationType" />
         </service>
+
+        <service id="sylius.form.type.attribute_type.select" class="Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\SelectAttributeType">
+            <argument>%locale%</argument>
+            <tag name="form.type" />
+        </service>
+
+        <service id="sylius.form.type.attribute_type.select.choices_collection" class="Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration\SelectAttributeChoicesCollectionType">
+            <argument id="sylius.translation_locale_provider" type="service"/>
+            <tag name="form.type" />
+        </service>
+
+        <service id="sylius.attribute_type.select.value.translations" class="Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration\SelectAttributeValueTranslationsType">
+            <argument id="sylius.translation_locale_provider" type="service"/>
+            <tag name="form.type" />
+        </service>
```

Comment:
I guess there's still something wrong with those definitions, `%locale%` points to a base locale code, whereas `sylius.translation_locale_provider` service has `getDefaultLocaleCode` method which returns current channel's default locale. Is there any reason for using these two simultaneously instead of only one of them?

Reply:
I didn't notice that 👍 
I've changed to use `sylius.translation_locale_provider` instead of `%locale%`.